include/deal.II/grid/grid_tools.h

```diff
@@ -974,6 +974,41 @@ namespace GridTools
   get_patch_around_cell(const typename MeshType::active_cell_iterator &cell);
 
 
+  /**
+   * This function runs through the degrees of freedom defined by the
+   * Container and for each dof constructs a vector of active_cell_iterators
```

Comment:
Since you've started work on this patch, @drwells has changed the commonly used notation in #2093 to use the word "Mesh" instead of "Container" both in the documentation and in the name of the template parameter. Can you adjust to this notation?